### ISSUE TITLE
Option to ignore either PR's or Issues

### ISSUE THREAD

**Issue by Safihre** [actions/stale#837]
The cpython project runs the stale action to only mark PR's as stale.
However, due to the huge amount of PR's it never gets near to the newer PR's because it runs out of operations.
See: https://github.com/python/cpython/blob/9151bbefea3fb932eb6aa6ddb22d64b83f8149c7/.github/workflows/stale.yml

The action could process many more PR's if the issues would not be checked at all, since no action is defined for them anyway.

Could there be a way to force the action to skip issues (or PR's, for someone that just wants to check issues)?

**Comment by jaredh** [comment]
The docs for [days-before-close](https://github.com/actions/stale#days-before-close) and [days-before-stale](https://github.com/actions/stale#days-before-stale) state setting the value to `-1` will prevent it from closing or labelling stale. There are issue specific overrides available called [days-before-issue-close](https://github.com/actions/stale#days-before-issue-close) and [days-before-issue-stale](https://github.com/actions/stale#days-before-issue-stale) to achieve this.

**Comment by Safihre** [comment]
I know, but that's not what this is about. It will not label them, but it will still _process_ them at the cost of API actions.
I want to prevent it from even looking at issues, so it can look at more PR's.

**Comment by Safihre** [comment]
No not stale. Just nobody cared to respond.

**Comment by steveluscher** [comment]
Duplicate of #792.

**Comment by Safihre** [comment]
Is it? That issue is not very clearly described as being the same problem. It might be, but then should be more clearly described.

**Comment by Safihre** [comment]
Yes, but for different underlying reasons. So a solution for one will most likely not resolve the other.

**Comment by kurtqq** [comment]
this is much needed one!
any chance it will be added in future version?

**Comment by testworksau** [comment]
I'd be keen to see this functionality too, in some repos I want issues to remain open forever, and PRs to be closed after they become stale.

**Comment by lowlydba** [comment]
Any updates here? 2026 checking in and this is an important feature to have. I have to imagine on GitHub's side there is *some* motivation to add a feature that could grossly reduce the number of throwaway API calls from this action, and reduce the runtime on GitHub hosted runners?

**Comment by artsiommiksiuk** [comment]
Same here. In this AI heavy politics environment and work optimization we strangely still can't have anything done, the same way as in 2010.
